docs/usage/self-hosted-configuration.md

```diff
@@ -217,6 +217,12 @@ The `gitAuthor` option accepts a RFC5322-compliant string.
 **Note** We strongly recommend that the Git author email you use is unique to Renovate.
 Otherwise, if another bot or human shares the same email and pushes to one of Renovate's branches then Renovate will mistake the branch as unmodified and potentially force push over the changes.
 
+## gitNoVerify
+
+Controls when Renovate passes the `--no-verify` flag to `git`. The flag can be passed to `git commit` and/or `git push`.
+Please refer to the documentation for [git commit](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---no-verify)
+and [git push](https://git-scm.com/docs/git-push#Documentation/git-push.txt---no-verify) to see exactly what this flag does.
```

Comment:
```suggestion
Read the documentation for [git commit --no-verify](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---no-verify) and [git push --no-verify](https://git-scm.com/docs/git-push#Documentation/git-push.txt---no-verify) to learn exactly what each flag does.
```

- Rewrite sentence a bit
- Put sentence on one line
- Put long command options in link name